frontend/src/components/Profile/ProfilePreview/ProfileMeta/RecruiterProfileMeta.tsx

```diff
@@ -17,6 +18,7 @@ export const RecruiterProfileMeta = React.memo<Props>((props) => {
           className={cn(styles.recruiterMetaItem, typography.smallHeading)}
           key={item.name}
         >
+          {item.name === 'city' && <IconMapLocation />}
```

Comment:
let's make it possible to add icon to every component and to not rely on item.name